### PR TITLE
DEV: Simplify tests for semantic segmentation engine

### PR DESCRIPTION
**Pull request by vqdang**
- Reduce the time for dummy tests by changing the resolutions.

### CODE REVIEW

File: tests/models/test_semantic_segmentation.py [hunk 1]
```diff
@@ -626,25 +634,24 @@ def __init__(self):
 def test_functional_pretrained(remote_sample, tmp_path):
     """Test for load up pretrained and over-writing tile mode ioconfig."""
     save_dir = pathlib.Path(f"{tmp_path}/output")
-    mini_wsi_svs = pathlib.Path(remote_sample("svs-1-small"))
+    mini_wsi_svs = pathlib.Path(remote_sample("wsi4_1k_1k_svs"))
     reader = get_wsireader(mini_wsi_svs)
     thumb = reader.slide_thumbnail(resolution=1.0, units="baseline")
-    thumb = thumb[1024:1536, 1024:1536, :]
```
Comment (shaneahmed) [inline tests/models/test_semantic_segmentation.py:632]: why not use a 512x512 image? Why stick to 1kx1k?

File: tests/models/test_semantic_segmentation.py [hunk 2]
```diff
@@ -707,7 +714,7 @@ def test_behavior_bcss_local(remote_sample, tmp_path):
     save_dir = pathlib.Path(tmp_path)
 
     _rm_dir(save_dir)
-    wsi_breast = pathlib.Path(remote_sample)
+    wsi_breast = pathlib.Path(remote_sample("wsi4_4k_4k_svs"))
```
Comment (shaneahmed) [inline tests/models/test_semantic_segmentation.py:717]: this is again a 4kx4k image.

**Review by shaneahmed — COMMENTED** [review]
@vqdang 
Please address the comments and we can merge then.

File: tests/models/test_semantic_segmentation.py (hunk 2 above)
Comment (vqdang) [reply tests/models/test_semantic_segmentation.py:717]: This test is local test and is skipped on travis so I dont think we need to change anything there.
You can see the
> @pytest.mark.skip(reason="Local manual test, not applicable for travis.")

on top of the function.

File: tests/models/test_semantic_segmentation.py (hunk 1 above)
Comment (vqdang) [reply tests/models/test_semantic_segmentation.py:632]: I uploaded 512x512.svs and used that now.

Comment (shaneahmed) [reply tests/models/test_semantic_segmentation.py:632]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/commit/0e9b817ff3c0ae08d9250f7da6f8a819f422f17a

**Review by shaneahmed — APPROVED** [review]
Thanks @vqdang

**Pull request merged**